dumpling/export/config.go

```diff
@@ -528,6 +534,28 @@ func ParseTableFilter(tablesList, filters []string) (filter.Filter, error) {
 	return filter.NewTablesFilter(tableNames...), nil
 }
 
+func GetConfTables(conf *Config, tablesList []string) (DatabaseTables, error) {
```

Comment:
do we need to add a UT for exported func?

Reply:
We can remove `conf` usage in this function because it's never used